spec/support/v5.2/sorbet_test_cases.rb

```diff
@@ -1,4 +1,4 @@
-# typed: true
+# typed: false
```

Comment:
This got changed automatically by running the specs though I'm not sure why.

Reply:
It's fine :-) It's auto-corrected by the spec I think. It's copied from generators/sorbet_test_cases and we override the sig when testing, so it doesn't matter the sig here